src/mem/corealloc.h

```diff
@@ -282,7 +281,7 @@ namespace snmalloc
       bumpptr = slab_end;
     }
 
-    ChunkRecord* clear_slab(Metaslab* meta, smallsizeclass_t sizeclass)
+    Metaslab* clear_slab(Metaslab* meta, smallsizeclass_t sizeclass)
```

Comment:
The return here was always just a type pun and can probably be done away with now.

Reply:
Good point. I had removed the use-site, but not the return.